featuretools/primitives/utils.py

```diff
@@ -423,8 +423,8 @@ def _haversine_calculate(lat_1s, lon_1s, lat_2s, lon_2s, unit):
 class HolidayUtil:
     def __init__(self, country='US'):
         try:
-            holidays.CountryHoliday(country)
-        except (KeyError, NotImplementedError):
+            holidays.country_holidays(country=country)
+        except NotImplementedError:
```

Comment:
@thehomebrewnerd this is the only other place that I found where we catch both exceptions. Is there another place you had in mind?

Reply:
Oh, I forgot they were all using the same `HolidayUtil` class. No, that't the only other place - sorry for the confusion.

Reply:
no worries, updated